portfolio/src/main/java/com/google/sps/servlets/RandomArtist.java

```diff
@@ -0,0 +1,47 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Random;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
+@WebServlet("/artist")
+public class RandomArtist extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      
+      // Declare and initialize ArrayList of favorite music artists
+      ArrayList<String> favArtists = new ArrayList<String>() {
+          {
+              add("Panic! At The Disco");
+              add("The Weeknd");
+              add("Anderson .Paak");
+              add("Hozier");
+              add("Daniel Caeasr");
+              add("Jacon Collier");
+              add("The Strokes");
+              add("Mana");
+              add("Lola Club");
+              add("Los Cafres");
+              add("Los Fabulosos Cadillacs");
+            }
+        };
```

Comment:
Optional/my thought process looking at this code:

I've actually never seen this way of adding items to an ArrayList. I would have done
```
ArrayList<String> favArtists = new ArrayList<String>();
favArtists.add("Panic! At The Disco");
favArtists.add("The Weeknd");
...
favArtists.add("Los Fabulosos Cadillacs");
```
I looked up how you did it, and looks legit, so you don't have to do anything with this comment, just wanted to point out what I would have done instead.

More of a neat trick than anything: Also you could do `List<String> favArtists = new ArrayList<String>()` - this is an interview trick I used to do less work overall, because if you wanted to switch from an ArrayList to a LinkedList, you'd just have to update the second part. Also works with Maps and HashMap/HashTable. But again, no need for you to actually do anything with this comment, just sharing some thoughts :)

Reply:
That trick is really cool I didn't know that! Thanks!